client/components/search/search-collection.jsx

```diff
@@ -0,0 +1,88 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+
+const Hider = React.createClass( {
+	displayName: 'Hider',
+
+	propTypes: {
+		hide: React.PropTypes.bool,
+	},
+
+	render() {
+		return (
+			<div
+				className={ 'design-assets__group' }
+				style={ this.props.hide ? { display: 'none' } : {} }
+			>
+				{ this.props.children }
+			</div>
+		);
+	}
+} );
+
+const FilterSummary = React.createClass( {
+	render() {
+		if ( this.props.items.length === 0 ) {
+			return ( <p>No results found.</p> );
```

Comment:
@dereksmart needs translation or be converted into something that can be changed by setting a property and add translation in Jetpack